### PR TITLE
sg: fix `live` command after ci build version update

### PR DESCRIPTION
**Pull request by leonore**
I think [this](https://github.com/sourcegraph/sourcegraph/pull/48050) broke `sg live` parsing, as it adds a release tag to the `__version` output afaict. 

## Test plan

```bash
$ cd dev/sg

$ go run . live s2
✅ Fetched deployed version
✅ Done updating list of commits

💡 Live on "s2": ff45c0f95a51 (built on 2023-02-23)

$ go run . live scaletesting
✅ Fetched deployed version
✅ Done updating list of commits

💡 Live on "scaletesting": 4cd778cfb5ab (built on 2023-02-15)

❗️ Deployed SHA 4cd778cfb5ab not found in last 20 commits on origin/main :(
```

### CODE REVIEW

**Comment by burmudar** [comment]
This it is a side effect from this recent change https://github.com/sourcegraph/sourcegraph/pull/48050

**Comment by burmudar** [comment]
This it is a side effect from this recent change https://github.com/sourcegraph/sourcegraph/pull/48050

**Review by burmudar — APPROVED** [review]
Thanks for the fix 🚀

File: dev/sg/live.go [hunk 1]
```diff
@@ -72,13 +72,20 @@ func printDeployedVersion(e environment, commits int) error {
...
 		return errors.Errorf("unknown format of /__version response: %q", body)
 	}
 
 	buildDate := elems[1]
-	buildSha := elems[2]
+
+	// split the release tag from the commit Sha
+	versionTag := strings.Split(elems[2], "-")
+	if len(versionTag) != 2 {
+		return errors.Errorf("unknown format of /__version tag: %q", elems[2])
```
[the remark this answers is not in the document]
Comment (leonore) [reply dev/sg/live.go:86]: good catch, yeah seems reasonable to me 😄

**Pull request merged**